cmd/mt-update-ttl/main.go

```diff
@@ -58,89 +36,73 @@ func init() {
+	flag.StringVar(&cfg.Timeout, "cassandra-timeout", cfg.Timeout, "cassandra timeout")
+	flag.IntVar(&cfg.WriteConcurrency, "cassandra-concurrency", 20, "number of concurrent connections to cassandra.") // this will launch idle write goroutines which we don't need but we can clean this up later.
+	flag.IntVar(&cfg.Retries, "cassandra-retries", cfg.Retries, "how many times to retry a query before failing it")
+	flag.IntVar(&cfg.WindowFactor, "window-factor", cfg.WindowFactor, "size of compaction window relative to TTL")
+	flag.IntVar(&cfg.CqlProtocolVersion, "cql-protocol-version", cfg.CqlProtocolVersion, "cql protocol version to use")
+	flag.BoolVar(&cfg.CreateKeyspace, "create-keyspace", cfg.CreateKeyspace, "enable the creation of the keyspace and tables")
+	flag.BoolVar(&cfg.SSL, "cassandra-ssl", cfg.SSL, "enable SSL connection to cassandra")
+	flag.StringVar(&cfg.CaPath, "cassandra-ca-path", cfg.CaPath, "cassandra CA certificate path when using SSL")
+	flag.BoolVar(&cfg.HostVerification, "cassandra-host-verification", cfg.HostVerification, "host (hostname and server cert) verification when using SSL")
+	flag.BoolVar(&cfg.Auth, "cassandra-auth", cfg.Auth, "enable cassandra authentication")
+	flag.StringVar(&cfg.Username, "cassandra-username", cfg.Username, "username for authentication")
+	flag.StringVar(&cfg.Password, "cassandra-password", cfg.Password, "password for authentication")
+	flag.StringVar(&cfg.SchemaFile, "schema-file", cfg.SchemaFile, "File containing the needed schemas in case database needs initializing")
+	flag.BoolVar(&cfg.DisableInitialHostLookup, "cassandra-disable-initial-host-lookup", cfg.DisableInitialHostLookup, "instruct the driver to not attempt to get host info from the system.peers table")
+
+	cfg.ReadConcurrency = 0
+	cfg.ReadQueueSize = 0
+	cfg.WriteQueueSize = 0
+
+	flag.IntVar(&startTs, "start-timestamp", 0, "timestamp at which to start, defaults to 0")
+	flag.IntVar(&endTs, "end-timestamp", math.MaxInt32, "timestamp at which to stop, defaults to int max")
+	flag.IntVar(&numThreads, "threads", 10, "number of workers to use to process data")
+	flag.IntVar(&statusEvery, "status-every", 100000, "print status every x keys")
+
+	flag.BoolVar(&verbose, "verbose", false, "show every record being processed")
+
 	flag.Usage = func() {
 		fmt.Fprintln(os.Stderr, "mt-update-ttl [flags] ttl table-in [table-out]")
 		fmt.Fprintln(os.Stderr)
 		fmt.Fprintln(os.Stderr, "Adjusts the data in Cassandra to use a new TTL value. The TTL is applied counting from the timestamp of the data")
 		fmt.Fprintln(os.Stderr, "If table-out not specified or same as table-in, will update in place. Otherwise will not touch input table and store results in table-out")
-		fmt.Fprintln(os.Stderr, "In that case, it is up to you to assure table-out exists before running this tool")
+		fmt.Fprintln(os.Stderr, "Unless you disable create-keyspace, table-out is created when necessary")
```

Comment:
This assumes that table-out is named using the internal naming convention of MT.  But there are no checks to ensure that is true.

Reply:
i suppose we could check if the table generated for the requested TTL matches the name specified.
but at that point, we may as well get rid of the table-out argument and just always generate it automatically. mt-update-ttl then knows the right destination table without further user input, and even know when to do updates (if it's same as table-in) or copies (if it's different). does that sound good to you @woodsaj ?

Reply:
That seems reasonalbe, but you should then do the same for both the input and out tables. ie `old-ttl`, `new-ttl`